SYCL/ESIMD/lsc/Inputs/lsc_usm.hpp

```diff
@@ -0,0 +1,108 @@
+//==------------ lsc_usm.cpp - DPC++ ESIMD on-device test ------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
```

Comment:
I suggest to handle this test same way as you did for other tests. 
I.e. have all the content of the main test in lsc_usm.cpp (not in this "Inputs" folder), and the other 64-bit version of the test would just include that lsc_usm.cpp